src/Generator/GeneratorCommand.php

```diff
@@ -84,6 +87,18 @@ public function getDefaultFileName(): string
         return 'Default' . $this->getFileTypeCapitalized();
     }
 
+    protected function setOptions(): void
```

Comment:
This needs to consider a specific yaml file in the project root, which is git ignored.